docker-compose.yml

```diff
@@ -70,6 +70,12 @@ services:
       - CORPPASS_IDP_ID
       - IS_SP_MAINTENANCE
       - IS_CP_MAINTENANCE
+      - SGID_ENDPOINT
+      - SGID_CLIENT_ID
+      - SGID_CLIENT_SECRET
+      - SGID_REDIRECT_URI
+      - SGID_PRIVATE_KEY
+      - SGID_PUBLIC_KEY
```

Comment:
heads up - I _think_ we are running against the maximum length of 4096 characters for environment variables in Elastic Beanstalk. have you tried deploying to staging?

Reply:
we haven't, given that there's still a bunch of setup to undertake. Thanks for notifying!

Reply:
... do we have a viable solution to the extensive configuration that FormSG needs? I wouldn't want to compete with the [Java runtime for number of env vars supported](http://bash.org/?14351)